README.md

```diff
@@ -49,11 +49,12 @@ This will uncompress the file and converts it to avro, leaving the original file
 For a full list of available options, run `csv2avro --help`
 ```
 $ csv2avro --help
-Version 1.0.1 of CSV2Avro
+Version 1.3.0 of CSV2Avro
```

Comment:
Well, that's the next upcoming version, we're at `1.2.0` now. I just thought I'd bump, it doesn't really matter.